client/components/status/status.js

```diff
@@ -149,8 +150,9 @@ export default class Status extends React.Component {
         options.currentPage = params.page;
       }
       if (
-        "link" in response.headers &&
-        !response.headers.link.includes("next")
+        ("link" in response.headers &&
+          !response.headers.link.includes("next")) ||
+        !("link" in response.headers)
```

Comment:
can you explain me what is going on here? Is this a fix for a bug? If yes how to replicate it and why there's no test for it?

Reply:
It's a fix for a bug. 
When a new user just signups and log in and log out for few times (suppose 2 times) then the past session length is 2 which is less than the pagination length defined in openwisp-radius i.e. 10. So the API doesn't include `link` in headers because there is no next for past sessions. 
But according to the previous condition, it will not set hasMoreSession to `false` which is required to stop infinite loading of past sessions. So it will fetch the next page which results in an error and automatically log out.

I will write a test for this.